types/module/module.go

```diff
@@ -484,7 +495,11 @@ func (m *Manager) EndBlock(ctx sdk.Context, req abci.RequestEndBlock) abci.Respo
 	validatorUpdates := []abci.ValidatorUpdate{}
 
 	for _, moduleName := range m.OrderEndBlockers {
-		moduleValUpdates := m.Modules[moduleName].EndBlock(ctx, req)
+		module, ok := m.Modules[moduleName].(EndBlockAppModule)
+		if !ok {
+			continue
+		}
+		moduleValUpdates := module.EndBlock(ctx, req)
```

Comment:
```suggestion
		module, ok := m.Modules[moduleName].(EndBlockAppModule)
		if ok {
			moduleValUpdates := module.EndBlock(ctx, req)
		}
```

Reply:
Not sure if this is possible as `moduleValUpdates` is used outside of if statement

Reply:
ahh right, my bad 👍